runtime/tests/org.eclipse.core.tests.runtime/src/org/eclipse/core/tests/runtime/AutomatedRuntimeTests.java

```diff
@@ -31,7 +31,7 @@
 		XmlProcessorFactoryTest.class, //
 		/*
 		 * Intentional the LAST TEST in the list to let
-		 * JobEventTest.testNoTimeoutOccured() verify the other tests:
+		 * JobEventTest.testNoTimeoutOccurred() verify the other tests:
```

Comment:
As this is method name I went to check if the method name is changed too but there is no such test method name at all. Please remove the whole comment as it's useless now.